backend/src/session/session.module.ts

```diff
@@ -0,0 +1,31 @@
+import * as ConnectRedis from 'connect-redis';
+import * as session from 'express-session';
+import { NestSessionOptions, SessionModule } from 'nestjs-session';
+
+import { IConfig, ConfigService } from '@config';
+import { Redis, RedisService } from '@redis';
+
+const RedisStore = ConnectRedis(session);
+
+export const Session = SessionModule.forRootAsync({
+	imports: [ Redis ],
+	inject: [ RedisService, ConfigService ],
+	useFactory: (redisService: RedisService, config: ConfigService<IConfig>): NestSessionOptions => {
+		const redisClient = redisService.getClient();
+		const store = new RedisStore({ client: redisClient, logErrors: true });
+		return {
+			session: {
+				store,
+				secret: config.get<string>('api.session.secret' as keyof IConfig),
+				resave: true,
+				saveUninitialized: true,
+				cookie: {
+					secure: false,
+					sameSite: false,
+					httpOnly: true,
+					maxAge: 60000 * 60 * 6,
```

Comment:
Please consider using an ENV variable for this.

Reply:
Fixed in https://github.com/Cleopha/ifttt-like/pull/1/commits/bcbba3317a65cfb2873f47a75ea3928771a9d94c